components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa/src/Publisher/Models/OpcNodeOnEndpointModel.cs

```diff
@@ -0,0 +1,46 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.IIoT.OpcUa.Publisher.Models {
+    using System;
+    using Newtonsoft.Json;
+    using System.Text;
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// Class describing a list of nodes
+    /// </summary>
+    [DataContract]
+    public class OpcNodeOnEndpointModel {
+
+        /// <summary> Node Identifier </summary>
+        [DataMember(EmitDefaultValue = false, IsRequired = false)]
+        public string Id { get; set; }
+
+        /// <summary> Expanded Node identifier </summary>
+        [DataMember(EmitDefaultValue = false, IsRequired = false)]
+        public string ExpandedNodeId { get; set; }
+
+        /// <summary> Sampling interval </summary>
```

Comment:
Please specify whether interval is in seconds or milliseconds for `OpcSamplingInterval`, `OpcPublishingInterval` and `HeartbeatInterval` as they are different in `OpcNodeModel`.

Reply:
added